frontend/src/components/atoms/SlideInDrawer.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box } from '@chakra-ui/react';
+import { AnimatePresence } from 'framer-motion';
+import React from 'react';
+import { MotionBox } from '../../types/react/componentTypes';
+
+interface SlideInDrawerProps {
+  expanded: Boolean;
```

Comment:
```suggestion
  expanded: boolean;
```

Should be primitive type if I'm not mistaken